python/cudf/dataframe/datetime.py

```diff
@@ -161,6 +164,16 @@ def to_arrow(self):
             null_count=self.null_count
         )
 
+    def default_na_value(self):
+        """Returns the default NA value for this column
+        """
+        dkind = self.dtype.kind
+        if dkind == 'M':
+            return np.datetime64('nat', 'ms')
```

Comment:
Just a note, we'll have to update this in the future as we add support for additional timestamp resolutions.

Reply:
do we have an issue to track needing to support additional units?

Reply:
https://github.com/rapidsai/cudf/issues/140

Reply:
Vintage :)